frontend/assignments/1/index.css

```diff
@@ -0,0 +1,807 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+@import url("https://cdn-uicons.flaticon.com/2.1.0/uicons-solid-rounded/css/uicons-solid-rounded.css");
+@import url("https://cdn-uicons.flaticon.com/2.1.0/uicons-brands/css/uicons-brands.css");
+@import url("https://cdn-uicons.flaticon.com/2.1.0/uicons-regular-rounded/css/uicons-regular-rounded.css");
+@import url("https://cdn-uicons.flaticon.com/2.1.0/uicons-thin-rounded/css/uicons-thin-rounded.css");
+@import url("https://cdn-uicons.flaticon.com/2.1.0/uicons-thin-straight/css/uicons-thin-straight.css");
+@import url("https://cdn-uicons.flaticon.com/2.1.0/uicons-bold-straight/css/uicons-bold-straight.css");
+@import url("https://cdn-uicons.flaticon.com/2.1.0/uicons-regular-rounded/css/uicons-regular-rounded.css");
+@import url('https://cdn-uicons.flaticon.com/2.1.0/uicons-solid-rounded/css/uicons-solid-rounded.css');
+@import url('https://cdn-uicons.flaticon.com/2.1.0/uicons-bold-rounded/css/uicons-bold-rounded.css');
+@import url('https://cdn-uicons.flaticon.com/2.1.0/uicons-regular-straight/css/uicons-regular-straight.css');
+
+* {
+  background-color: black;
+  margin: 0;
+  padding: 0;
+  color: white;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 15px;
+}
+
+svg {
+  fill: rgb(239, 243, 244);
+}
+
+textarea:focus {
+  outline: none;
+}
+
+.dp,
+.dp1,
+.dp2,
+.dp3 {
+  border-radius: 50%;
+}
+
+.lower {
+  height: 54px;
+}
+
+.lower-container {
+  display: flex;
+  height: 54px;
+  border-bottom: 1px solid rgb(47, 51, 54);
+}
+
+.for-you {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 53px;
+}
+
+.following {
+  height: 53px;
+}
+
+.for-you,
+.following {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-weight: 700;
+}
+
+.post-container {
+  padding: 0 16px;
+}
+
+.middle-post-box {
+  display: flex;
+  flex-direction: column;
+  height: 225px;
+  padding-top: 4px;
+  box-sizing: border-box;
+}
+
+.middle-post-box-1 {
+  display: flex;
+  height: inherit;
+  width: inherit;
+  flex-direction: column;
+}
+
+.middle-post-box-1-1 {
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.middle-post-box-1-1-left {
+  padding-top: 12px;
+  height: inherit;
+  box-sizing: border-box;
+  margin-right: 12px;
+}
+
+textarea {
+  height: inherit;
+  width: inherit;
+  border: 0;
+  font-size: 20px;
+}
+
+.middle-post-box-2-1 {
```

Comment:
use better naming conventions. They must denote the logic encapsulated instead of plain numerical suffix.
use BEM naming convention